### DISCUSSION TITLE
A Travel AP

### DISCUSSION THREAD

**Discussion by feh123** [RaspAP/raspap-webgui#731] [Ideas]
Hopefully in 2021 we can start doing things like travelling! When I travel I always take my RaspAP (RPi 4). Sometimes the router is in a really bad position so I take +10m of ethernet cable too. The reason I take it is that I have devices that use it's AP and I don't want to have to write new wpa_supplicant.conf files everytime I get to a new location. 
I have a RaspAP (RPi +3B) that runs wlan0/wlan1 with a wifi dongle attached. 
What would be great would be to take this second dongle (forget the ethernet!) and just "send it" the new SSID and password so it would work in my new location. My devices would then use the specific wifi.
How to "send it" - I would like this to be very easy! Maybe just via my phone? Any thoughts appreciated.

**Comment by GiannisDorokidis** [comment]
I am using the same setup as you but I have a Rpi zero w + wlan1 (usb dongle) . You can power up your pi connect with your phone to its own Ap go to the web-gui and add a wifi network from there
click on add and connect then reboot your pi. This should work fine.

**Comment by feh123** [comment]
Thanks @GiannisDorokidis for this. Covid restrictions give me few opportunities to try this out! I was thinking I could try this at home. 

So if I go into the GUI and alter the SSID and password (altering the SSID should be enough?) to some arbitrary wifi and reboot the RPi. It will now not be on my network so I will have no IP address to log into. It will still broadcast it's wifi? If I connect my Tablet or iPhone to that wifi I will get an IP address? If I go to that IP address I can add back the correct SSID and password for my home wifi? Is that correct? 

Sorry for the extra questions! Thanks for helping.

**Comment by GiannisDorokidis** [comment]
@feh123 
>It will still broadcast it's wifi?

Yes

>connect my Tablet or iPhone to that wifi I will get an IP address? If I go to that IP address I can add back the correct SSID and password for my home wifi? Is that correct?

Yes

If your are not sure if this is going to work I suggest you put in your Phone's hotspot ssid and password and you can still connect to it by using the ip its showing on your phone

**Comment by feh123** [comment]
@GiannisDorokidis thanks for the fast answers. That works - with my iPhone on the RPi AP hotspot I could connect to 10.3.141.1 to bring up the web-gui. I could scan and clearly I could have connected to any network in range! Thanks a lot for this! Hopefully soon I can go and try it out for real!
